test/package/bundleVersionCreate.test.ts

```diff
@@ -0,0 +1,761 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import path from 'node:path';
+import fs from 'node:fs';
+import { expect } from 'chai';
+import { Connection, SfProject } from '@salesforce/core';
+import { instantiateContext, restoreContext, stubContext, MockTestOrgData } from '@salesforce/core/testSetup';
+import { Duration } from '@salesforce/kit';
+import { PackageBundleVersion } from '../../src/package/packageBundleVersion';
+import { PackageBundleVersionCreate } from '../../src/package/packageBundleVersionCreate';
+import { BundleVersionCreateOptions, BundleSObjects } from '../../src/interfaces';
+
+// Type for accessing private methods in tests
+interface PackageBundleVersionCreateWithPrivates {
```

Comment:
you can remove this type and use ts-directives to access private members/methods in tests if you want
like `@ts-ignore-next-line` or whatever is required